### PR TITLE
Fix ra candidate creation with institution selection

### PR DESCRIPTION
**Pull request by pablothedude**
In order to be able to determine which ra candidates vould be made raa for which institution we need to return the institutions to this change will make sure this is done.

### CODE REVIEW

File: src/Surfnet/StepupMiddlewareClientBundle/Identity/Dto/RaCandidateInstitution.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
```
Comment (MKodde) [inline src/Surfnet/StepupMiddlewareClientBundle/Identity/Dto/RaCandidateInstitution.php]: Please adjust the licence headers for the two newly created files

2019 and SURFnet B.V.

File: src/Surfnet/StepupMiddlewareClientBundle/Identity/Dto/RaCandidateInstitution.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace Surfnet\StepupMiddlewareClientBundle\Identity\Dto;
+
+use Surfnet\StepupMiddlewareClientBundle\Dto\Dto;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class RaCandidateInstitution implements Dto
+{
+    /**
+     * @Assert\NotBlank(message="middleware_client.dto.institution_listing.institution.must_not_be_blank")
```
Comment (MKodde) [inline src/Surfnet/StepupMiddlewareClientBundle/Identity/Dto/RaCandidateInstitution.php]: The translation key refers to another DTO (institution listing).

Comment (pablothedude) [reply src/Surfnet/StepupMiddlewareClientBundle/Identity/Dto/RaCandidateInstitution.php]: Fixed

File: src/Surfnet/StepupMiddlewareClientBundle/Identity/Dto/RaCandidateInstitution.php (hunk 1 above)
Comment (pablothedude) [reply src/Surfnet/StepupMiddlewareClientBundle/Identity/Dto/RaCandidateInstitution.php]: updated

**Pull request merged**